src/bos/common/options.py

```diff
@@ -29,6 +29,7 @@
 # code should either import this dict directly, or (preferably) access
 # its values indirectly using a DefaultOptions object
 DEFAULTS = {
+    'pcs_transition_deadline': 60,
```

Comment:
The only concern I have is that I notice we are defaulting to 60 minutes for the deadline. Prior to this PR, our calls do PCS did not specify a deadline, meaning it would end up using the PCS default value of 5 minutes.

With a deadline of 60 minutes, for a node which is not behaving nicely with PCS, this could result in BOS creating a bunch of transitions in PCS for the same node. I don't know enough about how PCS works to know how it handles this situation, or if there is the potential for problems.

Thinking about it now, it almost seems like it would make more sense for us to specify a deadline that is no larger than the time left before BOS gives up on the node, right? Like if we're trying to power on a node, and there's only 4 more minutes before BOS is going to timeout and power off the node to retry, then it seems like our power on call shouldn't have a deadline more than 4 minutes (since a transition after that point doesn't help BOS anyway, right?).

It doesn't mean we couldn't also have an option like this, but it seems like we'd want to take the lower of the two values -- our time left before timeout and whatever is specified by this option.

Does that make sense?